release/_releases_json.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 
+function _add_new_version {
+	if [ -n "${ADD_VERSION%/*}" ]
```

Comment:
Hi @natocesarrego these variables are in different scripts. Now it runs with the `rebuild_releases_json.sh` but you linked `build_release.sh` and `release_gold.sh` which are separately running scrips. So where should I put this change? I should ask because I don't know during which running process do we need to add a new version to the lists.
Thank you